dcor/_energy.py

```diff
@@ -28,29 +34,34 @@ def _energy_distance_from_distance_matrices(
     average: Callable[[ArrayLike], float]
         A function that will be used to calculate an average of distances.
         This defaults to np.mean.
-    stat_type: Literal['u', 'v']
-        If 'u', calculate energy distance using
+    stat_type: Union[str, EstimationStatistic]
+        If EstimationStatistic.USTATISTIC, calculate energy distance using
         Hoeffding's unbiased U-statistics. Otherwise, use von Mises's biased
-        V-statistics
+        V-statistics.
+        If this is provided as a string, it will first be converted to
+        an EstimationStatistic enum instance.
     """
+    if isinstance(stat_type, str):
+        stat_type = EstimationStatistic(stat_type)
```

Comment:
I usually use `upper` to accept also the lowercase variant of the string (although I know that this could be problematic in some locales, like Turkish. One more reason to prefer enums).

```suggestion
        stat_type = EstimationStatistic(stat_type.upper())
```